proto/greptime/v1/ddl.proto

```diff
@@ -163,10 +164,10 @@ message ModifyColumnType {
 }
 
 message ChangeTableOptions {
-    repeated ChangeTableOption change_table_options = 1;
+    repeated AlterOption change_table_options = 1;
 }
 
-message ChangeTableOption {
+message AlterOption {
   string key = 1;
   string value = 2;
 }
```

Comment:
Rename it to `TableOption` in https://github.com/GreptimeTeam/greptime-proto/pull/201

Reply:
It seems the k-v pair can be used anywhere. I used it in AlterDatabaseOptions as well.